index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7bfe064e41.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio Page</title>
+</head>
+<body>
+ <header>
+ <h1 class="my_logo"><a href="#">Atril</a></h1>
+ <nav>
+    <div class="bar"></div>
+    <div class="bar"></div>
+    <div class="bar"></div>
+ </nav>
+ </header>
+<section class="container">
+<section class="header-text">
+    <h2>I’m Atril <br>
+        Glad to see you!</h2>
+  <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+</section>
+
+<section class="contact_list">
+    <h3><a href="#">LET’S CONNECT</a></h3>
```

Comment:
- kindly, you are using the default `a` text color, please use `color: #7F8CFF;
`  to fit the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=NVsQsZj0ugMKgNNU-0).👌👌